auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/model/Entitlement.java

```diff
@@ -108,5 +108,6 @@ public enum Entitlement {
     CDP_ROTATE_SALTUSER_PASSWORD,
     CDP_USERSYNC_ENFORCE_GROUP_MEMBER_LIMIT,
     CDP_DATA_LAKE_BACKUP_RESTORE_PERMISSION_CHECKS,
-    CDP_DATAHUB_EXPERIMENTAL_SCALE_LIMITS
+    CDP_DATAHUB_EXPERIMENTAL_SCALE_LIMITS,
+    CDP_DATALAKE_BACKUP_ON_RESIZE_LONG_TIMEOUT
```

Comment:
This entitlement is to support backups that take longer. This is not related to resizing.

Reply:
fixed